data/build.gradle

```diff
@@ -82,12 +82,12 @@ dependencies {
       'androidx.test.ext:junit:1.1.1',
       'com.google.dagger:dagger:2.24',
       'com.google.truth:truth:1.1.3',
-      'com.squareup.okhttp3:mockwebserver:4.1.0',
-      'com.squareup.okhttp3:okhttp:4.1.0',
+      'com.squareup.okhttp3:mockwebserver:4.9.1',
+      'com.squareup.okhttp3:okhttp:4.9.1',
       'junit:junit:4.12',
       'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.2.2',
       'org.mockito:mockito-core:2.19.0',
-      'org.robolectric:robolectric:4.4',
+      'org.robolectric:robolectric:4.5',
```

Comment:
Is this because 4.5 is when SDK 30 classes were added, or for another reason (in which case, which)?

Reply:
Yes, SDK 30 is added in 4.5

Reply:
Please make sure that this is detailed in the PR description, too.

Reply:
Added